src/Illuminate/Mail/Mailable.php

```diff
@@ -461,6 +461,18 @@ public function replyTo($address, $name = null)
     }
 
     /**
+     * Determine if the given recipient is set on the mailable.
```

Comment:
The docs here should say "Determine if the given replyTo is set on the mailable." 

They are not a recipient of the mailable.